tutorials/export/exporting_for_android.rst

```diff
@@ -100,6 +100,7 @@ There are two types of icons required by Godot:
 
 - **Main Icon:** The "classic" icon. This will be used on all Android versions up to Android 8 (Oreo), exclusive. Must be at least 192×192 px.
 - **Adaptive Icons:** Starting from Android 8 (inclusive), `Adaptive Icons <https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive>`_ were introduced. Applications will need to include separate background and foreground icons to have a native look. The user's launcher application will control the icon's animation and masking. Must be at least 432×432 px.
+- **Themed Icons:** Starting from Android 13(inclusive), Themed Icons were introduced to make adaptive icons more customizable. Applications will need to include monochrome icon to enable themed icon. If monochrome icon is not provided default godot monochrome icon will be used insted.
```

Comment:
```suggestion
- **Themed Icons:** Starting from Android 13 (inclusive), Themed Icons were introduced to make adaptive icons more customizable. Applications will need to include a monochrome icon to enable themed icon. If a monochrome icon is not provided the default Godot monochrome icon will be used insted.
```

Reply:
I have squashed all commits now. I was having some internet issues, that's why it took some time.